package.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Should we add an author as well?

Reply:
Yes, we should do that. I can add "Helpshift" as the author.

Reply:
Cool

Reply:
Keeping this conversation open until I submit a commit to solve it

Reply:
Added author and license